ansible/roles/vendor/requirements.yml

```diff
@@ -14,18 +14,19 @@
   version: 1.1.0
   src: https://github.com/avanov/ansible-galaxy-pyenv.git
 - name: davidwittman.redis
-  version: 1.2.6
+  src: https://github.com/adborden/ansible-redis.git
+  version: fix-initscript
```

Comment:
Maybe pin this to upstream at https://github.com/DavidWittman/ansible-redis/commit/98bef17f323438dec378527e459e7fd72d27e31f  instead of your fork?

Reply:
https://github.com/DavidWittman/ansible-redis/issues/224

Reply:
It hadn't been merged at the time of authoring, but now it's even in a release! https://github.com/DavidWittman/ansible-redis/releases/tag/1.2.7

Reply:
https://github.com/GSA/datagov-deploy/pull/955